core/templates/dev/head/domain/topics_and_skills_dashboard/TopicsAndSkillsDashboardBackendApiService.js

```diff
@@ -16,15 +16,27 @@
  * @fileoverview Service to retrieve information of topics and skills dashboard
   from the backend.
  */
+ oppia.constant(
+   'MERGE_SKILL_URL', '/merge_skill');
 
 oppia.factory('TopicsAndSkillsDashboardBackendApiService', [
-  '$http', function($http) {
+  '$http', 'MERGE_SKILL_URL', function($http, MERGE_SKILL_URL) {
     var _fetchDashboardData = function() {
       return $http.get('/topics_and_skills_dashboard/data');
     };
 
+    var _mergeSkills = function(oldSkillId, newSkillId) {
```

Comment:
Not sure about the naming, the URL says merge_skill and the function name is mergeSkills

Reply:
Done, renamed all to `MERGE_SKILLS`.